src/components/List/List.tsx

```diff
@@ -119,11 +119,6 @@ function List<T extends Item>({
     }
   }, [items, Array.from(selection.values())]);
 
-  const allSelectableSelected = useMemo(
```

Comment:
Looks like this was the only usage of `useMemo()`. Remove its import above to make the linter warnings will go away.